resources/lang/en/groups.php

```diff
@@ -182,4 +182,7 @@
   'no_other_nearby_events' => 'There are currently no other upcoming nearby events.',
   'postcode' => 'Group postcode',
   'groups_postcode_small' => '(Optional)',
+  'duplicate' => 'That group name already exists.  If it\'s yours, please go to the Groups page using the menu and edit it.',
```

Comment:
Can we change the first part to 'That group name (:name) already exists.' so the user sees what they entered.

Reply:
I think it's already visible in the form below the message, isn't it?

Reply:
Fair enough.  Added.